### DISCUSSION TITLE
Review camera detections?

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#15068] [Ask A Question] [answered]
I have detection set up for my camera to detect 'person'. It works fairly well but sometimes it detects a false positive.

Is there a way to review what exactly it thinks is a person? All I've found is older videos online that show this where it'll show a box around what it detected along with a percentage score of it being a person, but the interface in those videos is older and I can't seem to find where to get that info now (if it's even still an option).

I'm using the Frigate add-on; I can send my YAML if necessary.

**Comment by NickM-27** [comment, marked as answer]
This has been added for 0.15 called the object lifecycle view

**Reply by templeowls21** [reply]
Gotcha, thanks!
